node/sc/vt_recovery.go

```diff
@@ -18,12 +18,16 @@ package sc
 
 import (
 	"github.com/klaytn/klaytn/accounts/abi/bind"
-	"github.com/klaytn/klaytn/contracts/bridge"
 	"github.com/pkg/errors"
 	"sync"
 	"time"
 )
 
+const (
+	filterLogsStride = 100
+	maxPendingTxs    = 100000
+)
```

Comment:
How about enabling to set these parameters to unit test this feature?

Reply:
Thanks. Added a unit test.